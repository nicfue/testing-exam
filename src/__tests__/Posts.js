import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Posts from '../components/Posts';

test.skip('renders the app', () => {
  render(<Posts />);
});


it.skip('should have a post', () => {
  const wrapper = shallow(<Posts  />)
})
