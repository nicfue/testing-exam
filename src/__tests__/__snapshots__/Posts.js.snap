// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing posts component snapshot of comment 1`] = `
<div
  className="flex flex-wrap mx-auto pt-8"
>
  <CreateNewPost
    author="Zac"
    updatePosts={[Function]}
  />
</div>
`;
